src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt

```diff
@@ -300,4 +301,17 @@ class IndexingService @Inject constructor(
             throw IndexingException(ioException)
         }
     }
+
+    private fun continueOnExceptions(objectType: ZoekObjectType, fn: () -> Boolean): Boolean {
+        var currentFailureCount = failureCounters[objectType] ?: 0
+        return try {
+            runTranslatingToIndexingException { fn() }.also {
+                failureCounters[objectType] = 0
+            }
+        } catch (indexingException: IndexingException) {
+            failureCounters[objectType] = ++currentFailureCount
+            logTypeError(objectType, currentFailureCount, indexingException)
+            currentFailureCount < MAX_SEQUENTIAL_ERRORS
```

Comment:
This failure count feels a bit wrong because it greatly depends on the context. E.g. in the case of `reindexAllZaken` I think we should not fail on e.g. 20 exceptions at all. I think we should never fail in this case. Or at least not at this level. When reindexing all zaken (or taken etc) there can in theory be an infinite list of zaken throw an exception during indexing and we should still continue.

What is the problem you are trying to solve this such a max failure count? I have a feeling the current solution has wrong consequences.

I would also be in favour of merging the PR as we approved it (without a max failure count) and then adding this afterwards, if needed, as a separate PR because now I don't know what I approved anymore.. ;-)

Reply:
The problem is that if zrcClientService.listZaken throws an exception for some reason we end in endless loop. Not sure we want this code merged at all.

That's why I ended with the idea to have 20 sequential errors abort the loops we now have.
If one operation in between succeeds we're back at 0 errors.

Reply:
Trying to rewrite parts of the logic to better accommodate for ignoring exceptions